light-eventuate-4j/src/main/resources/templates/eventuate/hybrid/service/pom.xml.rocker.raw

```diff
@@ -11,10 +11,10 @@
     <properties>
         <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <version.light-4j>1.5.18</version.light-4j>
-        <version.light-hybrid-4j>1.5.18</version.light-hybrid-4j>
+        <version.light-4j>1.5.32</version.light-4j>
```

Comment:
not for this review but food for though - since 1.5.32 is not released, this should probably be 1.5.32-SNAPSHOT and should be uploaded to snapshot repository.

Reply:
It is always a good idea to have a snapshot version published and @NicholasAzar mentioned that before. There is still a lot of details we need to learn. Like where the snapshot version going to be published? How is maven going to figure out which version of the snapshot to be downloaded? What needs to be changed in the light-bot to support snapshot etc.